community_components.md

```diff
@@ -42,6 +42,12 @@ Master thesis which consists of a completely automated system that provides
 a web interface for configuring GrimoireLab, so that it analyizes any set of
 repositories. [More information (in Spanish)](http://zhquan.github.io/TFG/).
 
+* [Prosoul](https://github.com/Bitergia/prosoul)
+The goal of this project is to create a web editor and viewer for software quality models that are used to show metrics in a meaningful way. Also, importers and exporters for the different quality models used in the industry will be implemented so they can be used as models or evolved for the creation of new quality models. In the initial version the metrics supported will be the CROSSMINER ones (see below), but it will aso support GrimoireLab data. Based on the models, visualizations and assessment for the projects are generated. In the inital version visualizations will be created using GrimoireLab.
+
+* [Crossminer](https://github.com/crossminer) .
+The CROSSMINER Project, is a project funded under the European Union's Horizon 2020 Research and Innovation Programme to enable the monitoring, in-depth analysis and evidence-based selection of open source components, and facilitates knowledge extraction from large open-source software repositories.
```

Comment:
I think we need this as a separate pull request. I suggest that it is opened under a new heding "R&D Projects using GrimoireLab".

Reply:
Ok, thanks for the comments. I don't know if I should close this PR or what.. Any help is welcome

Reply:
If you find it appropriate, I would address the other comment (about Prosoul), and produce a new pr with this one. No need to close this pr, I would use it for the first text (Prosoul).